scenario/tasks.yaml

```diff
@@ -35,7 +35,9 @@ tasks:
    act_cleanup_images: ['CleanupImages']
    get_next_instance: ['GetInfoIter']
    rename_info_iter: ['RenameInfo', 'info_result', 'info']
+   rename_info_iter1: ['RenameInfo', 'info_result', 'info']
```

Comment:
Why do we need to make 2 instances here? We can just go forward only with 1 `rename_info_iter` here.